build.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
```

Comment:
@crazii As stated in the PR comment, this should ultimately be replaced with a step in the workflow that invokes `GPR2MAK` to generate a properly working Makefile from `sbemu.gpr` and then building SBEMU using that Makefile. But I'll need some help with that. See issue #19.

Reply:
It will be better if we write another makefile for Linux to build it natively, since DJGPP package is available on Linux too. Actually THP has done it.

Reply:
I know, but then we would have to maintain both the GPR file and the Makefile, since they could go out of sync from each other, if developers update one but forget the other. I was kind of hoping that we could have one Makefile or project file as the "single source of truth". Since you are (or were?) the main developer of SBEMU and apparently chose RHIDE as your development environment for this project, I wanted to respect that, even though I prefer to use a Makefile and a cross-compilation toolchain myself.

Reply:
I saw in [your response to thp's PR](https://github.com/crazii/SBEMU/pull/28#issuecomment-1817479662) that this won't be an issue anymore. That's good to know. Thanks.

@crazii Another outstanding question is how to build a compatible version of HDPMI that works with SBEMU and can be packaged into the USB image that this CI/CD pipeline would build. Is it just the HEAD of the master branch of [your HX fork](https://github.com/crazii/HX)? Or is it a specific branch or something? Thanks.

Reply:
Yes it's the HEAD. But I don't know how to build it on Linux. :D

Reply:
Thanks. Let me give it a try.

Reply:
I updated the PR, removing the case insensitive workaround and the now redundant `.build.sh` file, using @thp's makefile instead. The one thing missing from the USB image that gets created is the HDPMI executable.

@crazii Perhaps a reasonable intermediate solution would be if you created a release on https://github.com/crazii/HX with the HDPMI executable as a downloadable artifact? Then I can update this PR to just download that executable from that release page for now, and then we can look into setting up CI/CD for that project separately later.

Reply:
@crazii Do you know how to build it in DOS? Also, in the meantime, would DOSEMU work with the two build artifacts from March, or are they too old? Adding the correct version of HDPMI is the only remaining step for me to add to this PR. Once I've added that, the pipeline will automatically build a ready-to-use bootable USB image with FreeDOS, Jemm and SBEMU. Thanks!